File: maria-health/Maria Health Phi/fibonacci.js
function fibonacci(num) {
	var x = 0, y = 1;

	for (var i=0; i <= num; i++) {
		var z = x + y;
        x = y;
        y = z;

        if (z % 3 == 0 && z % 5 == 0) {
            z = "Maria Health";
        } else if (z % 3 == 0) {
            z = "Maria";
        } else if (z % 5 == 0) {
            z = "Health";
        }

        console.log(z)
	}


}

fibonacci(100)

File: maria-health/Maria Health Phi/fibonacci.py
def fibonacci(num):
    x, y = 0, 1
    for i in range(num):
        i += 1
        z = x + y
        x = y
        y = z

        if z % 3 == 0 and z % 5 == 0:
            z = "Maria Health"
        elif z % 3 == 0:
            z = "Maria"
        elif z % 5 == 0:
            z = "Health"

        print(z)

fibonacci(20)